src/coreclr/jit/compiler.h

```diff
@@ -2023,6 +2025,9 @@ struct NaturalLoopIterInfo
     // Tree that mutates the induction variable.
     GenTree* IterTree = nullptr;
 
+    // Is the loop exited when TestTree is true?
+    bool ExitedOnTrue : 1;
```

Comment:
Since I spent time understanding the logic I added this now, even though it is always expected to be false currently. I expect to make a future change that removes the use of `GetLexicallyBottomMostBlock` in `AnalyzeIteration`, at which point this will be able to be true.